.rubocop.yml

```diff
@@ -1,29 +1,169 @@
+inherit_from:
+  - .rubocop_todo.yml
+
 AllCops:
+  TargetRubyVersion: 2.1
+  TargetRailsVersion: 3.2
   Include:
     - '**/Rakefile'
     - '**/config.ru'
   Exclude:
     - 'db/**/*'
     - 'config/**/*'
     - 'script/**/*'
-    - 'spec/**/*'
     - !ruby/regexp /old_and_unused\.rb$/
 
-Documentation:
+# OFN SETTINGS
+# Cop settings that have been agreed upon by the OFN community
+
+Rails:
+  Enabled: true
+
+Style/Documentation:
+  Enabled: false
+
+Style/StringLiterals:
+  Enabled: false
+
+# TEMPORARY/CONTESTED SETTINGS
+# These are still to be decided upon, but recommended for inclusion by
+# oeoeaio after scrutinising offenses the codebase
+
+# Don't think this is a big issue, mostly picking up RPSEC scope definitions
+# with lamdas and RSpec '.to change{}' blocks
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
+# No alternative to this one until we upgrade to Rails 4 and can use #find_by
+Rails/DynamicFindBy:
+  Enabled: false
```

Comment:
Yeah, I understand all that that, I guess my feeling was that when such a large number of files need to be excluded, the value in retaining the cop on the remaining files is substantially diminished, because it is almost random chance whether future violations will be caught or not. Happy to go with your approach though.

My one remaining issue with that approach for this particular cop is: what do we suggest people do instead? The Ruby style guide [suggests that](https://github.com/bbatsov/rails-style-guide#find_by) a `where(...).first` is just as undesirable as a dynamic `find_by_...`.

Further, I think that forcing developers to use `where(...).first` instead of dynamic matchers is a retrograde step, because switching to `find_by(...)` will probably be easier from dynamic matchers than it will be from `where(...).first`...